store/sqlstore/upgrade.go

```diff
@@ -18,7 +18,8 @@ import (
 )
 
 const (
-	CURRENT_SCHEMA_VERSION   = VERSION_5_20_0
+	CURRENT_SCHEMA_VERSION   = VERSION_5_21_0
```

Comment:
See earlier comment, but let's point this at v5.22, since v5.21 is exclusively in the release-5.21 branch.